api/src/main/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimRepository.java

```diff
@@ -18,11 +18,12 @@
 @JaversSpringDataAuditable
 public interface ClaimRepository extends CrudRepository<Claim, UUID>, ClaimLazyLoader {
 
-    @Query("SELECT COUNT(claim) "
+    @Query("SELECT claim.id "
             + "FROM Claim claim "
             + "WHERE claim.claimant.nino = :nino "
-            + "AND claim.claimStatus in ('NEW', 'ACTIVE', 'PENDING', 'PENDING_EXPIRY')")
-    Long countLiveClaimsWithNino(@Param("nino") String nino);
+            + "AND claim.claimStatus in ('NEW', 'ACTIVE', 'PENDING', 'PENDING_EXPIRY')"
+            + "ORDER BY claim.claimStatusTimestamp DESC")
+    List<UUID> findLiveClaimsWithNino(@Param("nino") String nino);
```

Comment:
Could we change this to return a boolean, e.g liveClaimsExistWithNino. seeing as we never actually use the contents.